### ISSUE TITLE
Fake ERRORs in DL2 to DL3 step

### ISSUE THREAD

**Issue by moralejo** [cta-observatory/cta-lstchain#1225]
When running the DL2 to DL3 step for a source-independent analysis, one gets this error if IRF interpolation is used:

2024-02-10 13:32:49,219 ERROR [lstchain.high_level.interpolate] (interpolate.interpolate_irf): AL CUTS not present for IRF interpolation
Obviously, Alpha cuts do not exist for this type of analysis, and the DL3 file is nevertheless produced (as it should).
If one uses explicitly the option 
--source-dep False

it is even worse... it throws the same error plus this one:
2024-02-10 13:28:32,031 ERROR [lstchain.DataReductionFITSWriter] (tool.run): Caught unexpected exception: File does not contain source-dependent parameters

The more you tell it you are doing a source-**in**dependent analysis, the more it wants to do it!

**Comment by SeiyaNozaki** [comment]
Yes... this is a fake error as reported also in #1180  and vice versa... (The script raises an error saying RAD_MAX is not present for IRF creation when using source-dependent analysis). Need to be fixed.

**Comment by moralejo** [comment]
Ok, I see. The same happens with the creation of the index files. It complains, for a point-like DL3 set, that the background and psf HDUs are missing...

2024-02-10 16:51:54,442 ERROR [lstchain.high_level.hdu_table] (hdu_table.create_hdu_index_hdu): Run 10946 does not contain HDU BACKGROUND
2024-02-10 16:51:54,442 ERROR [lstchain.high_level.hdu_table] (hdu_table.create_hdu_index_hdu): Run 10946 does not contain HDU PSF

**Comment by SeiyaNozaki** [comment]
Yes... I'm on it to solve this fake error
